### DISCUSSION TITLE
表单设计器联动x-reactions使用数组不能反向预览，会提示Property expected

### DISCUSSION THREAD

**Discussion by wangjie19910615** [alibaba/formily#2421] [Q&A 2.x] [answered]
"x-reactions": {[
          {
            "target": "target",
            "when": "{{$self.value === '123'}}",
            "fulfill": {
              "state": {
                "visible": false
              }
            }
          },
        ]},

**Comment by janryWang** [comment, marked as answer]
一定要基于响应器配置出来的结构才能回显

**Reply by wangjie19910615** [reply]
嗯，是我写错了，x-reactions后面不需要{}，直接[]就可以了，感谢大神！
